src/Arcus.Scripting.Tests.Unit/Arcus.Scripting.ApiManagement.tests.ps1

```diff
@@ -1013,5 +1013,310 @@ InModuleScope Arcus.Scripting.ApiManagement {
+
+                # Assert
+                Assert-VerifiableMock
+                Assert-MockCalled Get-AzApiManagement -Times 1
+                Assert-MockCalled Get-AzCachedAccessToken -Times 1
+                Assert-MockCalled Invoke-WebRequest -Times 1
+            }
+            It "Inviting a user in Azure API Management and include a note is OK" {
+                # Arrange
+                $resourceGroup = "contoso"
+                $serviceName = "contosoApi"
+                $firstName = "John"
+                $lastName = "Doe"
+                $mailAddress = "john.doe@contoso.com"
+                $stubApiManagement = New-Object -TypeName Microsoft.Azure.Commands.ApiManagement.Models.PsApiManagement
+
+                Mock Get-AzApiManagement {
+                    $ResourceGroupName | Should -Be $resourceGroup
+                    $Name | Should -Be $serviceName
+                    return $stubApiManagement } -Verifiable
+                Mock Invoke-WebRequest -MockWith {
+                    $status = [System.Net.WebExceptionStatus]::Success
+                    $response = New-Object -type 'System.Net.HttpWebResponse'
+                    $response | Add-Member -MemberType noteProperty -Name 'StatusCode' -Value 200 -force
+
+                    return $response
+                }
+                Mock Get-AzCachedAccessToken -MockWith {
+                    return @{
+                        SubscriptionId = "123456"
+                        AccessToken = "accessToken"
+                    }
+                }
+
+                # Act
+                Create-AzApiManagementUser `
+                    -ResourceGroupName $resourceGroup `
+                    -ServiceName $serviceName `
+                    -FirstName $firstName `
+                    -LastName $lastName `
+                    -MailAddress $mailAddress `
+                    -Note 'this is a note'
+
+                # Assert
+                Assert-VerifiableMock
+                Assert-MockCalled Get-AzApiManagement -Times 1
+                Assert-MockCalled Get-AzCachedAccessToken -Times 1
+                Assert-MockCalled Invoke-WebRequest -Times 1
+            }
+            It "Inviting a user in Azure API Management and specify a UserId is OK" {
+                # Arrange
+                $resourceGroup = "contoso"
+                $serviceName = "contosoApi"
+                $firstName = "John"
+                $lastName = "Doe"
+                $mailAddress = "john.doe@contoso.com"
+                $stubApiManagement = New-Object -TypeName Microsoft.Azure.Commands.ApiManagement.Models.PsApiManagement
+
+                Mock Get-AzApiManagement {
+                    $ResourceGroupName | Should -Be $resourceGroup
+                    $Name | Should -Be $serviceName
+                    return $stubApiManagement } -Verifiable
+                Mock Invoke-WebRequest -MockWith {
+                    $status = [System.Net.WebExceptionStatus]::Success
```

Comment:
A test that would return another HTTP status code, that wouldn't be any added-value here, right? Since we don't do anything with the response.
Should we, though, verify the response in the script? Otherwise we would have traces saying `Account has been created for $FirstName $LastName ($mailAddress)` while that wasn't the case.
But I'm not familiar enough with the possible return types here. Maybe we only need to verify if the response is expected.

Reply:
Did some manual testing and if I break the http request the script nicely shows me the error and doesn't output `Account has been created for $FirstName $LastName ($mailAddress)`. 
I don't think this needs further verification.

Reply:
Ok, super! Thx for checking 👍